config/env/staging.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
    ***************************************************************************/
 
   models: {
-    migrate: 'safe',
+    migrate: 'alter',
```

Comment:
@itsalaidbacklife this was key to getting postgres working on docker, but I'm not entirely sure why. I guess it allows it to alter an already established db but we'll need to chat through this.

Reply:
It's a setting that tells sails what to do when booting to make the database match what's expected. There are three options:
`safe` - preserve the database as-is and crash if data is misshapen
`alter` - leave existing rows but add necessary tables and columns where possible 
`drop` - delete everything and create all the tables from scratch 

Here are the [docs](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#migrate)